source/extensions/extensions_build_config.bzl

```diff
@@ -153,6 +153,7 @@ EXTENSIONS = {
     "envoy.tracers.datadog":                            "//source/extensions/tracers/datadog:config",
     "envoy.tracers.zipkin":                             "//source/extensions/tracers/zipkin:config",
     "envoy.tracers.opencensus":                         "//source/extensions/tracers/opencensus:config",
+    "envoy.tracers.skywalking":                         "//source/extensions/tracers/skywalking:config",
```

Comment:
Set this as WiP since in source/extensions/tracers/skywalking/BUILD we say the status is wip.